apps/condo/domains/billing/schema/BillingReceipt.js

```diff
@@ -53,6 +56,16 @@ const BillingReceipt = new GQLListSchema('BillingReceipt', {
             },
         },
 
+        category: {
+            schemaDoc: 'A category to print on display on the payment document.',
+            type: Relationship,
+            ref: 'BillingCategory',
+            isRequired: true,
+            knexOptions: { isNotNullable: true },
+            kmigratorOptions: { null: false, on_delete: 'models.PROTECT' },
+            defaultValue: { connect: { id: DEFAULT_CATEGORY } },
+        },
```

Comment:
should we also add a constraint: UNIQ: period + category + account ?

Reply:
As of now, I don't want to add any logic on categories

Reply:
And any restrictions? is it ok for third-party integrations?